modules/administration-guide/pages/enabling-fuse-for-all-workspaces.adoc

```diff
@@ -7,42 +7,19 @@
 [id="enabling-fuse-overlayfs-for-all-workspaces"]
 = Enabling fuse-overlayfs for all workspaces
 
+For Podman 5.x, the `/home/user/.config` must be owned by the current user for Podman to function correctly.
+The `storage.conf` file for Podman is typically stored in this folder.
+This document explains how to configure the workspace's container entrypoint script for the workspace so that fuse-overlayfs is being used for all workspaces using that container.
+The Universal Developer Image (UDI) already contains the necessary configuration by default.
+
 .Prerequisites
 
-* The xref:administration-guide:enabling-access-to-dev-fuse-for-openshift.adoc[] section has been completed. This is not required for OpenShift versions 4.15 and later.
+* For OpenShift versions 4.14 and lower, the xref:administration-guide:enabling-access-to-dev-fuse-for-openshift.adoc[] section has been completed.
 
 * An active `{orch-cli}` session with administrative permissions to the destination OpenShift cluster. See {orch-cli-link}.
```

Comment:
```suggestion
* An active `{orch-cli}` session with administrative permissions to the destination OpenShift cluster. See {orch-cli-link}.

* For OpenShift versions 4.14 and lower, the xref:administration-guide:enabling-access-to-dev-fuse-for-openshift.adoc[] section has been completed.

* For Podman 5.x, the `/home/user/.config` must be owned by the current user. This is where the `storage.conf` file for Podman is typically stored.
```

Reply:
> For Podman 5.x, the `/home/user/.config` must be owned by the current user. This is where the `storage.conf` file for Podman is typically stored.

This part is what the procedure aims to do, so it doesn't belong in the prerequisites. I've added a new commit to explain this part in the procedure steps: 5795e7c957cf887fa2d12048ed9fcace6ace36f8

WDYT?

Reply:
I see. sorry about that.

Reply:
your edit looks great. now I understand how it all works.